src/app/content/guards.ts

```diff
@@ -18,3 +22,16 @@ export const isLinkedArchiveTreeSection =
 
 export const hasOSWebData = (book: Book | undefined): book is BookWithOSWebData =>
   book ? 'slug' in book : false;
+
+export const isReferenceLoadingError = (something: any): something is ReferenceLoadingError =>
+  something instanceof ReferenceLoadingError;
+
+export const isContentPageRefencesType = (something: any): something is ContentPageRefencesType =>
+  isObject(something)
+  && typeof (something as ContentPageRefencesType).match === 'string'
+  && typeof (something as ContentPageRefencesType).pageId === 'string';
+
+export const isPageReferenceMapError = (something: any): something is PageReferenceMapError =>
```

Comment:
you can start with the interface from the state here instead of `any` and it'll probably be a lot cleaner 

i think maybe just `{match: string}` would be the place to start